cpplint.py

```diff
@@ -6740,8 +6735,8 @@ def PrintUsage(message):
   Args:
     message: The optional error message.
   """
-  sys.stderr.write(_USAGE  % (sorted(list(GetAllExtensions())),
-       ','.join(sorted(list(GetAllExtensions()))),
+  sys.stderr.write(_USAGE  % (sorted(GetAllExtensions()),
```

Comment:
double space at "_USAGE  " - odd that ruff isnt picking that up

Reply:
This will be automatically fixed when we add `ruff format` to pre-commit.